Entities/BookEntity.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Entities;
+
+class BookEntity
+{
+  private $id;
+  private $title;
+  private $description;
+  private $rating;
+  private $ISBN_13;
+  private $ISBN_10;
+  private $image;
+  private $language;
+  private $price;
+  private $currency;
+  private $buy_link;
+
+  public function __construct($params)
+  {
+    foreach ($params as $key => $value) {
+      $this->{$key} = $value;
+    }
+  }
+
+  public function getId()
+  {
+    return $this->id;
+  }
+
+  public function setId($id)
```

Comment:
Add documentation for each setter/getter with type of variable that is expected or return type.